filebeat/module/system/auth/_meta/fields.yml

```diff
@@ -24,6 +24,54 @@
       description: >
         The Unix user that this event refers to.
 
+    - name: pam
+      type: group
+      description: >
+        Fields specific to PAM messages.
+      fields:
+      - name: module
+        description: >
+          The name of the PAM module.
+      - name: service
+        description: >
```

Comment:
Empty description, this can be the reason why docs build is failing.

Reply:
Thanks